mne/epochs.py

```diff
@@ -308,8 +316,29 @@ def __init__(self, info, data, events, event_id=None, tmin=-0.2, tmax=0.5,
             else:
                 self.drop_log = drop_log
             events = events[selected]
-            if len(np.unique(events[:, 0])) != len(events):
-                raise RuntimeError('Event time samples were not unique')
+            unique_events, counts = np.unique(events[:, 0], return_counts=True)
+            if len(unique_events) != len(events):
+                if event_repeated == 'error':
+                    raise RuntimeError('Event time samples were not unique')
+                elif event_repeated == 'merge':
+                    new_events = events.copy()
+                    non_uniques = unique_events[counts > 1]
+                    for uu in non_uniques:
```

Comment:
uu -> ev

to indicate it's an event

Reply:
`uu` and `ev` are both acceptable for a one-liner `[uu for uu in blabla]`. (I might be inclined to `uu` but this is more of a personal taste from a matlab background)

But for any other case than a one-liner, both name choices are bad. `current_event` or something readable. Otherwise make it a function. or something with a really (really) short scope.